components/resources/ai_chat_ui_strings.grdp

```diff
@@ -429,6 +432,9 @@
   <message name="IDS_CHAT_UI_CHAT_QWEN_SUBTITLE" desc="A description for qwen model" formatter_data="webui=AiChat">
       General purpose chat, coding, and more
   </message>
+  <message name="IDS_CHAT_UI_CHAT_GEMMA_SUBTITLE" desc="A description for gemma model" formatter_data="webui=AiChat">
+    General purpose text and image understanding
+  </message>
```

Comment:
<img width="384" height="520" alt="Screenshot 2025-07-28 at 2 12 39 PM" src="https://github.com/user-attachments/assets/e4e35542-0adb-489f-97c3-ea6ba1bc9c97" />

Subtitle is missing in settings, I think you missed adding an entry in AIChatSettingsHelper::GetModelsWithSubtitles in browser/ai_chat/ai_chat_settings_helper.cc.